### ISSUE TITLE
bug(docker): images are inconsistently publishing multiplatform builds

### ISSUE THREAD

**Issue by JonZeolla** [Checkmarx/kics#6800]
### Expected Behavior

All images on docker hub are all multi-platform (amd64/arm64).

### Actual Behavior

Only some images on docker hub are multi-platform.

### Details

It appears the `-debian` images are meant to build and push on release via [this](https://github.com/Checkmarx/kics/blob/3b468f2dfbe1ec2a26d6e94fa0e83e7604258b35/.github/workflows/release-dkr-image.yml#L63C50-L76) github action workflow it is [broken](https://github.com/Checkmarx/kics/actions/workflows/release-dkr-image.yml).

Below is a screenshot of [this](https://hub.docker.com/r/checkmarx/kics/tags?page=1&name=-debian) showing the inconsistency.
<img width="539" alt="Screenshot 2023-11-14 at 1 33 35 PM" src="https://github.com/Checkmarx/kics/assets/1385510/81483ff5-1a32-4d3f-9b19-d6767d7b806c">

CC: @nunoocx and @gabriel-cx - are you manually building and pushing the images?

**Comment by kicsbot** [comment]
Please, follow the guideline for an issue title: 

For **bug**:

`bug(<scope>): <title starting with lowercase letter>`

For **query**:

`query(<platform>): <title starting with lowercase letter>`

For **feature request**:

`feat(<scope>): <title starting with lowercase letter>`

Thank you!
*KICS Team*

**Comment by JonZeolla** [comment]
Specifically https://github.com/Checkmarx/kics/actions/runs/6800401414/job/18500698437 and https://github.com/Checkmarx/kics/actions/runs/6800401399/job/18488854312 are failing the debian (and ubi8) build and push.  Previously it was failing the generate .md table

**Comment by kicsbot** [comment]
Deleting comment, please refresh the page...

**Comment by JonZeolla** [comment]
I can't repro locally; docker builds and runs fine for me on my mac arm64 system with both 1.20.5 (buster) and 1.21.4 (bullseye) base images.  But I can't build and `--load` multi-platform due to a docker limitation, so it's a partial test.

**Comment by gabriel-cx** [comment]
Hi @JonZeolla ,

Thank you for your inputs.
We did have an issue regarding building some images in the latest release and we build and push manually the ones that failed.
However, we already fixed the root cause of that problem, so, in the next release we will not face similar issue! 

_"Previously it was failing the generate .md table"_ regarding this, you don't need to worry. We are analyzing the root cause, but this problem alone is not affecting the build or push of any image.

Does this information clarify your doubts?
